frontend/src/components/VulnerableGpaComponent.tsx

```
@@ -0,0 +1,238 @@
+    const handleFormSubmit = (event: React.FormEvent) => {
+        event.preventDefault();
+        
+        const formData = new FormData(event.target as HTMLFormElement);
+        const data: any = {};
+        
+        formData.forEach((value, key) => {
+            data[key] = value;
+        });
+        
+        if (data.callback) {
+            eval(data.callback);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)